### ISSUE TITLE
Add option to select all duplicate files from same folder

### ISSUE THREAD

**Issue by ghost** [qarmin/czkawka#1324]
**Feature Description**
...
Sometimes it is useful to pick and delete the duplicates which exist in a specific folder.

I suggest to add an option to the "select" button to select all duplicates from folder of the selected duplicate entry.

**Comment by LordDrakath** [comment]
The select button at the bottom. You can choose to select custom and set the path of the folder you want selected.

**Comment by SamuelAllain** [comment]
Hello ! For me, this option would be very useful ! For instance i have a large folder with all my pictures. And I tidied some in smaller folders but I forgot to remove them from the large folder. Then Czkawka helps me removing these duplicates from the large folder.

> The select button at the bottom. You can choose to select custom and set the path of the folder you want selected.

I find that setting the path of the folder is too long for the operation I use the most. Thanks !

**Comment by AlexioXela** [comment]
> The select button at the bottom. You can choose to select custom and set the path of the folder you want selected.

This would be a life saver! Is that a suggestion, or are you saying that's how it works now? It doesn't show up for me. It looks like I'm using Krokiet version 9.0.0 on Windows 10, I don't know how parity between Krokiet and Czkawka is.

**Comment by saucefrog** [comment]
"select all files in this folder" should really be there as a direct option.

**Comment by Suncatcher** [comment]
Agree, it's easy to make it a separate button, not setting a path via custom selection. Why still not done?

**Comment by mvoggu** [comment]
This seems to be now implemented, as part of the Right click context menu added in https://github.com/qarmin/czkawka/pull/1888

It hasn't been included as part of any release yet though.
Below is a screenshot from Krokiet on windows built from the latest source code:

<img width="2559" height="843" alt="Image" src="https://github.com/user-attachments/assets/d0a6a1ee-794b-4aef-b77c-5656c27a48eb" />

**Comment by Suncatcher** [comment]
> This seems to be now implemented, as part of the Right click context menu added in [#1888](https://github.com/qarmin/czkawka/pull/1888)

I tried latest release 11.0.1 and I don't see this menu item. This PR wasn't yet released?

**Comment by qarmin** [comment]
It is available in master branch

**Comment by Gadgitmatic** [comment]
> > This seems to be now implemented, as part of the Right click context menu added in [#1888](https://github.com/qarmin/czkawka/pull/1888)
> 
> I tried latest release 11.0.1 and I don't see this menu item. This PR wasn't yet released?

Same. Mine opens parent folder when I right-click. No menu at all.

**Comment by Suncatcher** [comment]
> Same. Mine opens parent folder when I right-click. No menu at all.

works in Krokiet 12.0.0 by right click, I just verified.
@qarmin now the next logical step would be to enable selection of all dups in the file parent directory up to N levels up.
Example: I have a dir `nvme0n1p4` and I found a duplicate in the path `nvme0n1p4/OneDrive/Filebox/Dictionaries/Wiki`. I want Czkawka to jump up to 4 levels to directory `nvme0n1p4` and select **all duplicate files in all directories within `nvme0n1p4`** up to 4 levels deep
- `nvme0n1p4/OneDrive/dup.txt`
- `nvme0n1p4/OneDrive/Filebox/dup.txt`
- `nvme0n1p4/OneDrive/Filebox/Dictionaries/dup.txt`
The bove sample uses `dup.txt` file but I mean it should select such paths from all duplicate groups, not only in the same group.
<img width="1104" height="388" alt="Image" src="https://github.com/user-attachments/assets/ff2665e7-7fe2-4804-83c7-a7a45b3cba55" />
